testing.Makefile

```diff
@@ -0,0 +1,46 @@
+ifndef __TESTING_MAKEFILE__
+
+__TESTING_MAKEFILE__ := included
+
+include common.Makefile
+include gcloud.Makefile
```

Comment:
nit: this should be included at the top level instead, but let's make that change when we actually include this makefile. Can you add a TODO here?